### DISCUSSION TITLE
Renault Megane IV Touch Screen

### DISCUSSION THREAD

**Discussion by mikaba-eu** [cedricp/ddt4all#1131] [Q&A]
Have someone a Idea how I can fix my touchscreen? My workshop has installed the latest version of my RLINK. Since then, the touchscreen no longer works properly. I can only touch the top right-hand corner, where inputs are registered. After a few adjustments, I was able to get at least the entire right edge to work.

Can maybe someone share his settings?

I have the big screen (just a photo from the internet now):
![image](https://github.com/cedricp/ddt4all/assets/117845495/ae7d5b79-22a6-4e05-94b7-c7f9f460ac3f)

![3](https://github.com/cedricp/ddt4all/assets/117845495/cb0e02d0-2d05-4650-93d0-904a1ac8a288)
![2](https://github.com/cedricp/ddt4all/assets/117845495/43e7e22b-5a6b-4b6a-b083-6d9f4495b0b2)
![1](https://github.com/cedricp/ddt4all/assets/117845495/b336b88c-e3bb-4eb1-bbc2-b3517c37d8df)

**Comment by mikaba-eu** [comment]
Can someone share the original working settings? ...

**Reply by dominicusmento** [reply]
the picture you've given doesn't hold original big screen for megane IV.. you have original ?
